File: lib/euler-one.c
/* This program has been worked on from http://www.s-anand.net/euler.html */

#include <stdio.h>

sofmul()
{
int n, i;
n = 0;

for (i = 0 ; i < 1000 ; i++) {
  if  ( !(i % 5) ||  !(i % 3) )
    n = n + i;
 }

printf("Sum of natural numbers which are the multiples of 3 and 5 below 1000 is %d \n", n);
}

File: lib/euler-one.py
'''
This file is worked on from http://www.s-anand.net/euler.html  problem 1.

Find the sum of all the multiples of 3 or 5 below 1000 

'''

n = 0
for i in xrange(1,1000):
    if not i % 5 or not i % 3:
        n = n + i
print n
